hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/ContractFeature.java

```diff
@@ -131,6 +139,27 @@ public void verifyContractFunctionCallMirror() {
         verifyContractExecutionResultsByTransactionId();
     }
 
+    @Then("I call the contract via the mirror node REST API")
+    public void restContractCall() {
+        ContractCallResponse getAccountBalanceResponse = mirrorClient.contractsCall(GET_ACCOUNT_BALANCE_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress());
+        assertThat(convertContractCallResponseToNum(getAccountBalanceResponse)).isEqualTo(BigInteger.valueOf(1000L));
+
+        ContractCallResponse getSenderResponse = mirrorClient.contractsCall(GET_SENDER_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress());
+        assertThat(convertContractCallResponseToAddress(getSenderResponse)).isEqualTo(contractClient.getClientAddress());
+
+        ContractCallResponse multiplySimpleNumbersResponse = mirrorClient.contractsCall(MULTIPLY_SIMPLE_NUMBERS_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress());
+        assertThat(convertContractCallResponseToNum(multiplySimpleNumbersResponse)).isEqualTo(BigInteger.valueOf(4L));
+
+        ContractCallResponse identifierResponse = mirrorClient.contractsCall(IDENTIFIER_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress());
+        assertThat(convertContractCallResponseToSelector(identifierResponse)).isEqualTo(IDENTIFIER_SELECTOR);
+
+        try {
+            mirrorClient.contractsCall(WRONG_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress());
+        } catch (Exception e) {
+            assertThat(e.getCause().getMessage()).contains("400 Bad Request from POST");
+        }
```

Comment:
This doesn't fail if no exception.
```suggestion
        assertThatThrownBy(() -> mirrorClient.contractsCall(WRONG_SELECTOR, contractId.toSolidityAddress(), contractClient.getClientAddress())).getCause().isInstanceOf(SomeException.class).hasMessageContaining("400 Bad Request from POST");
```